BackEndApp/authv1/urls.py

```diff
@@ -3,8 +3,9 @@
 from . import views
 
 urlpatterns = [
-    path('login/', views.login, name='login'),
-    path('register/', views.register, name='register'),
-    path('logout/', views.logout, name='logout'),
-
+    path("login/", views.login, name="login"),
+    path("register/", views.register, name="register"),
+    path("logout/", views.logout, name="logout"),
+    path("forgot-password/", views.ForgotPassword, name="forgot-password"),
+    path("password-reset/", views.PasswordReset, name="password-reset"),
```

Comment:
Please follow the convention that was used before `forgot_password` is more "pythonic" way :)
Same for PasswordReset